cophee_test/compare_output

```diff
@@ -619,6 +619,32 @@ compare "phpdoc block comment",
    */
   """
 
+compare "single line comment",
+  '''
+  ###
+  # Great test
+  #
+  # @dataProvider provider_test
+  ###
+  trait A
+    x: ->
+      # Some rnadom comment
+      3
+  ''',
+  """
+  /**
+   * Great test
+   *
+   * @dataProvider provider_test
+   */
+  trait A {
+    function x() {
+      // Some rnadom comment
+      return 3;
+    }
+  }
+  """
```

Comment:
@werlex Could you expand this test. Somehow currently copheescript adds space between the comment block and method. Maybe in this test it will show up. I have also added the spaces, which it would be great to preserve too.
```suggestion
  '''
  ###
  # Great test
  #
  # @dataProvider provider_test
  ###
  trait A

    ###
    # Great test
    ###
    x: ->

      # Some random comment
      3

  ''',
  """
  /**
   * Great test
   *
   * @dataProvider provider_test
   */
  trait A {

    /**
     * Great test
     */
    function x() {
    
      // Some rnadom comment
      return 3;
    }
  }
  """
```

Reply:
@headbandno2 it is weird that I wrote test with spaces and it kept those spaces. No addition space were added after comment block before function. So it is weird, that we are getting different output than tests